File: python/basic13/returnCountGreaterY.py
arr = [100, 200, 300, 400, 500]
y = 3

def countGreaterY(arr, y):
    count = 0
    value = []
    for i in range(len(arr)):
        if arr[i] > y:
            count += 1
            value.append(arr[i])
    print(count, value)

countGreaterY(arr, y)

File: basic13/returnCountGreaterY.js
arr = [15, 2, 30, 4, 50];
y = 10

function countGreaterY(arr, y) {
    let count = 0;
    for(let i = 0; i < arr.length; i++) {
        if(arr[i] > y) {
            count++
            console.log(arr[i])
        }
    }
    console.log("total greater than y is " + count)
}

countGreaterY(arr, y);